BufferedSerial/Buffer/MyBuffer.h

```diff
@@ -130,7 +132,9 @@ class MyBuffer
 template <class T>
 inline void MyBuffer<T>::put(T data)
 {
-    _buf[_wloc++] = data;
+    _buf[_wloc] = data;
+    __DSB();

```

Comment:
Are these portable across the other compilers?

Reply:
Hope so. It's CMSIS as far as I know.